create_coda_files/create_coda_files.py

```diff
@@ -58,32 +64,48 @@ def _open_scheme(filepath):
     traced_json_output_path = args.traced_json_output_path
     coda_output_path = args.coda_output_path
     coding_scheme_path = args.coding_scheme_path
-    if args.auto_cleaner not in AUTO_CLEANERS.keys():
-        auto_cleaner = args.auto_cleaner
-        auto_cleaner = AUTO_CLEANERS[auto_cleaner]
+    if args.has_yes_no == 'True':
+        has_yes_no = True
+    elif args.has_yes_no == 'False':
+        has_yes_no = False
+    else:
+        raise Exception('<has-yes-no> only takes the strings "True" or "False"')
```

Comment:
argparse can be configured to do this for you (choices=["True", "False"])

Reply:
changed